definitions/benefit/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1720,5 +1720,10 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "shouldBeIssued", "AccessProfile": "caseworker-sscs-clerk", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "shouldBeIssued", "AccessProfile": "caseworker-sscs-registrar", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "shouldBeIssued", "AccessProfile": "caseworker-sscs-judge", "CRUD": "CRUD"},
-  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "shouldBeIssued", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"}
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "shouldBeIssued", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "CaseFieldID": "ftaRequestType", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "CaseFieldID": "ftaRequestTopic", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "CaseFieldID": "ftaRequestQuestion", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"},
+  {"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "CaseFieldID": "ftaCommunications", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"}
+  
```

Comment:
as per requirements 5/6, add the following:
{"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "CaseFieldID": "ftaCommunicationFilter", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"},
{"LiveFrom": "13/03/2025", "CaseTypeID": "Benefit", "CaseFieldID": "tribunalCommunicationFilter", "AccessProfile": "caseworker-sscs-superuser", "CRUD": "CRUD"}

Reply:
Done